Fixed handling errors while licensing image from the media gallery

sivaschenko opened the pull request:
### Description (*)
Fixed handling errors while licensing image from the media gallery

### Fixed Issues (if relevant)
Closes https://github.com/magento/adobe-stock-integration/issues/1344

sivaschenko committed:
magento/adobe-stock-integration#1344: Fixed licensing images from Media Gallery

chalov-anton reviewed (changes requested):
❌ QA Failed
_Uncaught TypeError: Cannot read property 'add' of undefined_ error message appears in dev console 
![cannot_read_add](https://user-images.githubusercontent.com/45624059/81703173-e3f7b380-9474-11ea-9d20-c7d313841ddd.png)

Nazar65 commented:
@chalov-anton yes, that because wrong namespace here https://github.com/magento/adobe-stock-integration/blob/2376054b0fd6bfe47bafeea6354753cb2162cc88/MediaGalleryUi/view/adminhtml/templates/image_details_standalone.phtml#L50

need to be with standalone prefix `standalone_media_gallery_listing.standalone_media_gallery_listing`

sivaschenko committed:
magento/adobe-stock-integration#1345: Fixed saving licensed image

sivaschenko committed:
magento/adobe-stock-integration#1345: Removed extra empty line

sivaschenko commented:
Fix was updated and includes the following changes:
* Fixed issue view accessing View Details panel after saving licensed image
* User will not need to provide the image name during licensing already downloaded image
* Error messages that may occur during licensing will now be displayed in the top message area instead of alert
* Added success message after image was licensed
* Buttons have been reordered on View details panel to match mockups

sivaschenko committed:
Merge branch '2.0.0-develop' into localization-test-fix

sivaschenko committed:
magento/adobe-stock-integration#1345: Removed spaces

gabrieldagama commented on AdobeStockImageAdminUi/view/adminhtml/web/js/action/saveLicensed.js:
```diff
@@ -27,26 +27,25 @@ define([
-                deferred.resolve(destinationPath);
-            }).fail(function (message) {
-                deferred.reject(message);
+        } else {
```
I would personally prefer an early return rather than the else statement, but both are fine.

sivaschenko replied:
Thanks, done!

gabrieldagama reviewed (dismissed):
Thanks for your PR @sivaschenko, it looks good!

sivaschenko committed:
magento/adobe-stock-integration#1345: Code review changes

sivaschenko committed:
Merge branch '2.0.0-develop' into localization-test-fix

chalov-anton commented:
👍  QA Passed

* Fixed issue view accessing View Details panel after saving licensed image ✔️ 
* User will not need to provide the image name during licensing already downloaded image ✔️ 
* Error messages that may occur during licensing will now be displayed in the top message area ✔️ instead of alert ✔️ 
* Added success message after image was licensed ✔️ 
* Buttons have been reordered on View details panel to match mockups ✔️

sivaschenko committed:
Merge branch '2.0.0-develop' into localization-test-fix

sivaschenko commented:
@magento run all tests

ghost commented:
Hi @sivaschenko, thank you for your contribution! </br>Please, complete [Contribution Survey](https://opensource.magento.com/survey/contribution/magento+adobe-stock-integration+1345), it will take less than a minute. </br>Your feedback will help us to improve contribution process.
